Automattic/jetpack
# VideoPress: fix video privacy in the UI

This PR fixes the issue when showing the privacy of the video. It doesn't show the current value.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: fix current privacy of the video in the UI

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPres dashboard
* Edit the video privacy by clicking on the Privacy button
* Confirm it changes right after selecting the new value
* Confirm the current value is dissable 
* Confirm it's persisten after a hard refresh

<img width="449" alt="Screen Shot 2022-10-03 at 18 52 25" src="https://user-images.githubusercontent.com/77539/193692616-c72c1aad-7133-41c3-879b-74edb25e97ee.png">
